kitipy/groups.py

```diff
@@ -173,11 +165,92 @@ def merge(self, *args: click.Group):
                 One or many source click.Groups you want to merge in the
                 current Group.
         """
-        click_ctx = _fake_click_ctx()
         for src in args:
-            for cmdname in src.list_commands(click_ctx):
-                cmd = src.get_command(click_ctx, cmdname)
-                self.add_command(cmd)  # type: ignore
+            for cmd in src.commands.values():
+                self.add_command(cmd)
+
+            if isinstance(src, Group):
+                for group in src.transparent_groups:
+                    self.add_transparent_group(group)
+
+    def is_enabled(self, click_ctx: click.Context) -> bool:
+        """Check if the that Task should be filtered out based on click Context.
+        Most generally, you shouldn't have to worry about this method, it's 
+        automatically called by kitipy Group.
+
+        Args:
+            click_ctx (click.Context):
+                The click Context passed to the underlying filter.
+        
+        Returns:
+            bool: Either this task should be filtered in (True) or
+            filtered out (False).
+        """
+        for filter in self.filters:
+            if not filter(click_ctx):
+                return False
+        return self.hidden != True
+
+    def _resolve_commands(self, click_ctx: click.Context):
+        if len(self._resolved) > 0:
+            return self._resolved
+
+        commands = self._filter_command_list(click_ctx, self)
+        if len(commands) > 0:
+            names, origs, cmds = zip(*commands)
+        else:
+            names, origs, cmds = ((), (), ())
+
+        origins = dict(zip(names,
```

Comment:
> Is this 80 chars long rule not obsolete today? We don't have CRT screens with 800x600 resolution anymore.

Right, but 80 is still a great limit to keep readable code on splitted screens and avoids putting too much symbols and each line. 

> Can't you configure yapf to bypass this rule? If not then nevermind ¯_(ツ)_/¯

I'm gonna change yapf config but first I'd like to merge all the open PRs to avoid big diffs and rebases.